torch/distributed/distributed_c10d.py

```diff
@@ -4297,6 +4298,175 @@ def _get_backend_from_str(backend: Optional[str] = None) -> Backend:
     return Backend(backend)
 
 
+@_time_logger
+def split_group(
+    parent_pg=None,
+    split_ranks=None,
+    timeout=None,
+    pg_options=None,
+    group_desc=None,
+):
+    """
```

Comment:
should document that some backends do not support this API and what the behavior is in unsupported case